code/datums/components/crafting/recipes.dm

```diff
@@ -381,9 +381,31 @@
 				/obj/item/storage/firstaid = 1,
 				/obj/item/assembly/prox_sensor = 1,
 				/obj/item/bodypart/r_arm/robot = 1)
+	parts = list(/obj/item/storage/firstaid = 1,
+				/obj/item/healthanalyzer = 1)
 	time = 40
 	category = CAT_ROBOT
 
+/datum/crafting_recipe/medbot/on_craft_completion(mob/user, atom/result)
+	var/mob/living/simple_animal/bot/medbot/bot = result
+	var/obj/item/storage/firstaid/FA = bot.contents[3]
```

Comment:
Avoid single-or-double-upper-case-letter variable names.
`first_aid` or something like that works better.